### PR TITLE
add publication date to document search

### PR DESCRIPTION
**Pull request by annatraussnig**
The proof is in the pudding:

> Screenshot 2021-12-01 at 15 45 31

### CODE REVIEW

File: components/editor/utils/RepoSearch.js [hunk 1]
```diff
@@ -98,6 +114,28 @@ export const filterRepos = gql`
...
+    repo.latestCommit.document.meta.title ||
+    repo.id.replace([GITHUB_ORG, REPO_PREFIX || ''].join('/'), '')
+  const publicationDate =
+    repo?.latestPublications[0]?.document.meta.publishDate ||
+    repo.meta.publishDate
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      {title}
+      <br />
+      <small>
```
Comment (patrickvenetz) [inline components/editor/utils/RepoSearch.js:128]: Maybe use styleguide `Label`?

File: components/editor/utils/RepoSearch.js [hunk 2]
```diff
@@ -98,6 +114,28 @@ export const filterRepos = gql`
   }
 `
 
+const RepoItem = withT(({ t, repo }) => {
+  const title =
+    repo.latestCommit.document.meta.title ||
+    repo.id.replace([GITHUB_ORG, REPO_PREFIX || ''].join('/'), '')
+  const publicationDate =
+    repo?.latestPublications[0]?.document.meta.publishDate ||
```
Comment (patrickvenetz) [inline components/editor/utils/RepoSearch.js:122]:
```suggestion
    repo.latestPublications[0]?.document.meta.publishDate ||
```

(Assume `repo` is an object.)

**Review by patrickvenetz — APPROVED** [review]
I like shiny new things.

File: components/editor/utils/RepoSearch.js (hunk 1 above)
Comment (annatraussnig) [reply components/editor/utils/RepoSearch.js]: wise you are.

**Pull request merged**